doc/cygnus-common/installation_and_administration_guide/management_interface_v1.md

```diff
@@ -41,6 +41,11 @@ Content:
     * [GET `/v1/admin/metrics`](#section7.1)
     * [DELETE `/v1/admin/metrics`](#section7.2)
 * [Available aliases](#section8)
+* [Name Mappings](#section9)
```

Comment:
I think we have an specific .md for name mapings. A link to this part in management_interface_v1.md should be included in that document also.

Reply:
In this documente there are a links to  cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md

Reply:
What about the reverse link? I mean from cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md to this document. 

Something like: "NameMapping can be managed using a REST API, part of the general Cygnus administration API. The details can be found at (link)".

Reply:
Fixed in the followup PR #1568